src/wmtk/operations/utils/UpdateVertexMultiMeshMapHash.cpp

```diff
@@ -0,0 +1,95 @@
+#include "UpdateVertexMultiMeshMapHash.hpp"
+#include <wmtk/EdgeMesh.hpp>
+#include <wmtk/Mesh.hpp>
+#include <wmtk/PointMesh.hpp>
+#include <wmtk/SimplicialComplex.hpp>
+#include <wmtk/TetMesh.hpp>
+#include <wmtk/TriMesh.hpp>
+#include <wmtk/multimesh/utils/tuple_map_attribute_io.hpp>
+
+namespace wmtk::operations::utils {
+void update_vertex_operation_hashes(Mesh& m, const Tuple& vertex, Accessor<long>& hash_accessor)
```

Comment:
why not implement this as a static member function of the MultiMeshManager class to avoid having to friend this?

Reply:
updated, I made them a static member functions of multimesh manager and create a wrapper function for them in Mesh class.